pyglmnet/utils.py

```diff
@@ -83,28 +56,8 @@ def label_binarizer(y):
 
 def log_likelihood(y, yhat, distr):
     """Helper to compute the log likelihood."""
-    eps = np.spacing(1)
-    if distr in ['softplus', 'poisson']:
-        return np.sum(y * np.log(eps + yhat) - yhat)
-    elif distr == 'binomial' or distr == 'probit':
-        # Log likelihood of model under consideration
-        return 2 * len(y) * \
-            np.sum(y * np.log((yhat == 0) + yhat) / np.mean(yhat) +
-                   (1 - y) * np.log((yhat == 1) +
-                                    1 - yhat) / (1 - np.mean(yhat)))
-    elif distr == 'gaussian':
-        return np.sum((y - yhat)**2)
-    elif distr == 'gamma':
-        nu = 1.
-        return np.sum(nu * (-y / np.log1p(np.exp(yhat)) -
-                      np.log(np.log1p(np.exp(yhat)))))
-    elif distr == 'multinomial':
-        y = label_binarizer(y)
-        # yhat is the probability of each output
-        if yhat.ndim != y.ndim:
-            msg = 'yhat and ynull must be (n_samples, n_class) ndarrays'
-            raise ValueError(msg)
-        return np.sum(y * np.log(yhat))
+    from .pyglmnet import _logL
```

Comment:
ok, didn't catch the circular imports issue. right now, it's not meant to be exposed. however, i've seen several papers report log likelihood for model comparison instead of deviance. can we provide it as a score method? for now, i'm happy to hide it from the user in any way you choose.